README.md

```diff
@@ -62,17 +62,26 @@ More ML algorithms in cuML and more ML primitives in ml-prims are being worked o
 
 ## Installation
 
-Ensure `libomp` and `libopenblas` are installed, for example via apt:
+1. Install NVIDIA drivers with CUDA 9.2 or 10.0 
+2. Ensure `libomp` and `libopenblas` are installed, for example via apt:
 ```bash
 sudo apt install libopenblas-base libomp-dev
 ```
 
 #### Conda
 cuML can be installed using the `rapidsai` conda channel:
+
+CUDA 9.2
 ```bash
-conda install -c nvidia -c rapidsai -c conda-forge -c pytorch -c defaults cuml
+conda install -c nvidia -c rapidsai -c conda-forge -c pytorch -c defaults cuml cudatoolkit=9.2
```

Comment:
```suggestion
conda install -c nvidia -c rapidsai -c conda-forge cuml cudatoolkit=9.2
```

We can remove `pytorch` channel now, and I think we can also remove `defaults`

Reply:
We can't remove defaults as it has the conda compilers which conda forge no longer has as they'll depend on defaults.